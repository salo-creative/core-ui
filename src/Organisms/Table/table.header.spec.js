import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../test';

// COMPONENT
import TableHeader from './index';

test('Renders against snapshot', async () => {
  const { container } = renderWithTheme(
    <TableHeader />
  );
  expect(container.firstChild).toMatchSnapshot();
});