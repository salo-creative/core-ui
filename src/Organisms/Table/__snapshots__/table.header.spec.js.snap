// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders against snapshot 1`] = `
.c0 {
  background-image: linear-gradient( 90deg,rgba(255,255,255,0) 0,rgba(255,255,255,.8) 50%,rgba(255,255,255,0) 100% ),linear-gradient(rgb(225,225,225) 25px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 25px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 25px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(225,225,225) 15px,transparent 0),linear-gradient(rgb(255,255,255) 100%,transparent 0);
  background-size: 200px 100%,120px 25px,250px 15px,100px 15px,140px 15px,120px 25px,250px 15px,100px 15px,140px 15px,120px 25px,250px 15px,100px 15px,140px 15px,100% 100%;
  background-position: -150% 0,0px 0px,220px 5px,620px 5px,780px 5px,0px 55px,220px 60px,620px 60px,780px 60px,0px 110px,220px 115px,620px 115px,780px 115px,0 0;
  background-repeat: no-repeat;
  -webkit-animation: joXbPK 2.5s infinite;
  animation: joXbPK 2.5s infinite;
  width: 100%;
}

.c0 .salo-table__header-row,
.c0 .salo-table__body-wrapper {
  visibility: hidden;
}

.c1 {
  display: none;
  width: 100%;
  height: 5rem;
  -webkit-flex-wrap: none;
  -ms-flex-wrap: none;
  flex-wrap: none;
  border-bottom: 2px solid #e9ecef;
  position: relative;
}

.no-borders .c1 {
  border-bottom: none;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  height: 5rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 1.6rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2.error {
  color: #f15b60;
}

@media only screen and (max-width:0px) {
  .c0 {
    -webkit-animation: fHNosZ 2.5s infinite;
    animation: fHNosZ 2.5s infinite;
    background-position: -150% 0,0px 10px,0px 55px,0px 90px,0px 125px,0px 200px,0px 245px,0px 280px,0px 315px,0px 400px,0px 445px,0px 480px,0px 515px,0 0;
  }
}

<div
  class="c0 salo-table   salo-table--is-mounting"
  width="100%"
>
  <div
    class="c1 salo-table__header salo-table__header-row"
  />
  <div
    class="c2 salo-table__no-data"
  >
    There are no results to display
  </div>
</div>
`;
